Sources/Login/LoginCoordinator.swift

```diff
@@ -58,6 +58,9 @@ final class LoginCoordinator: NSObject,
         do {
             tokenHolder.accessToken = try userStore.secureStoreService.readItem(itemName: .accessToken)
             finish()
+        } catch is SecureStoreError {
```

Comment:
The only kind of error readItem can throw is a SecureStoreError, so no need for two catch blocks.  They can be combined as one.  It will never be a "Local Authentication error", that labeling is incorrect.

Reply:
Agree this should be a catch all I think, unless cancelling the local auth prompts throws an error..? I don't think I've tested that... we wouldn't want any action in that case, would just want to return and the user to be able to try again

Reply:
With some testing of this, I've had to change the first catch to `catch SecureStoreError.unableToRetrieveFromUserDefaults` and then have a general catch all below. With a general catch all, or for any `SecureStoreError` a returning user on the unlock screen cancelling Touch ID would be sent to the sign in page

Reply:
Do we need to handle `cantDecryptData` thrown by `decryptDataWithPrivateKey` in the same catch block as `unableToRetrieveFromUserDefaults`?